src/Tools/ExternalAccess/VisualDiagnostics/Contracts/HotReloadDocumentDiagnostics.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.Contracts
+{
+    internal class HotReloadDocumentDiagnostics(DocumentId documentId, ImmutableArray<Diagnostic> diagnostics)
```

Comment:
which Diagnostic type is this?  Is this the ROslyn LSP protocol type, or a different one?

Reply:
`Microsoft.CodeAnalysis.Diagnostic`